CHANGES.md

```diff
@@ -10,6 +10,7 @@ Next (unreleased)
 - Regions plugin:
   - Removed 'col-resize' cursor when resize is disabled (#1985)
   - Improved and unified loop playback logic (#1868)
+  - Check `minLength` before resizing region (#2001)
```

Comment:
please move it to the 'Next' section, 4.0.0 is already released.

Reply:
not sure what you mean, it's under Next (unreleased) for me?

Reply:
sorry, my mistake.